src/sentry/wsgi.py

```diff
@@ -20,6 +21,12 @@
 # Run WSGI handler for the application
 application = WSGIHandler()
 
+# our settings.LANGUAGE_CODE is 'en-us', but during requests it always
+# resolves to 'en', as 'en-us' is not a by default supported language.
+# activate the supported language variant, so the resolver warms up for
+# what serves most requests
+activate(get_supported_language_variant(settings.LANGUAGE_CODE))
+
```

Comment:
rather than calling this for a side-effect -- could we include the header which would trigger this in the request that's made?  that'd be more natural to the execution lifecycle imo

Reply:
i can do this, but during the request lifecycle, the language is not activated until after the URL Resolver has resolved the URL of the request, so the resolver populates `en-us` language on the request still..

I can add a `reverse` call within the `warmup` endpoint which would then populate the resolver with the `en` language. lmk if that is preferred.

Reply:
(this is because we call the `resolve` function in the `wsgi.py` file before any middleware has been ran)

Reply:
yeah that probably makes the most sense -- to make it more natural to understand why it's happening